### ISSUE TITLE
NullPointerExeption for Entity with no aspects set

### ISSUE THREAD

**Issue by TabakaSIM** [Project-RT/RandomTweaker#71]
I use `mods.randomtweaker.thaumcraft.IEntity.getAspects(entity)` to get entity aspects list. However if the entity doesn't have any set aspects (like zombie villager) it shows this error:
```
java.lang.NullPointerException
	at ink.ikx.rt.api.mods.thaumcraft.expand.ExpandIEntity.getAspects(ExpandIEntity.java:21)
	at do\Mithminite_scythe.vitreusCrystalize(do/mithminite_scythe.zs:741)
	at do\Mithminite_scythe.scytheEffectElemental(do/mithminite_scythe.zs:794)
```
Is there a way to return empty list for this case?

**Comment by ikexing-cn** [comment]
You can use `isNull` to handle with it, sry for now I don't have the free time to work on this issue now, but PR welcome.

**Comment by Krutoy242** [comment]
`isNull` can't be used here, because exception happen in Java code *before* any value returned to ZenScript.

**Comment by ikexing-cn** [comment]
Fixed, [#be30b31](https://github.com/Project-RT/RandomTweaker/commit/be30b31bf832907c9b0c87514566edfc928dd2fc)
